packages/medusa/src/services/order.ts

```diff
@@ -534,42 +535,25 @@ class OrderService extends TransactionBaseService {
    */
   async retrieveByExternalId(
     externalId: string,
-    config: FindConfig<Order> = {}
+    options: FindConfig<Order> = {}
```

Comment:
suggestion: not for that pr, but shouldn't we have
- protected retrieve_
- public retrieve call retrieve_
- public retrieveByCartId call retrieve_
- public retrieveByCartIdWithTotals call retrieveByCartId and apply totals
- public retrieveByExternalId call retrieve_
- public retrieveByExternalIdWithTotals call retrieveByExternalId and apply totals

In order to centralise the method logic?